packages/helpwave_widget/lib/src/static_progress_indicator/static_progress_indicator.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_theme/constants.dart';
+
+/// A Circular Progress indicator
+class StaticProgressIndicator extends StatelessWidget {
+  final Size size;
+
+  final double strokeWidth;
+
+  /// The semanticsLabel for Screen Reading Software
+  final String? semanticsLabel;
+
+  /// Progress as a Percentage between 0.0 to 1.0
+  final double progress;
+
+  /// Color that indicates Progress
+  final Color? color;
+
+  /// Color that is the background of the ring
+  final Color? backgroundColor;
+
+  const StaticProgressIndicator({
+    super.key,
+    required this.progress,
+    this.size = const Size.square(iconSizeSmall),
+    this.strokeWidth = 8,
+    this.semanticsLabel,
+    this.color,
+    this.backgroundColor,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: size.height - strokeWidth,
+      width: size.width - strokeWidth,
+      child: CircularProgressIndicator(
+        strokeWidth: strokeWidth,
+        semanticsLabel: semanticsLabel,
+        color: color ?? Theme
```

Comment:
use valueColor instead of color

Reply:
`Animation<Color?>?` is the type of the `valueColor`, which will then be a `ColorTween` because`Animation<Color?>?` is abstract.
`ColorTween` takes two colors, but we always have only one static color in the current design, which would reduce immediate readability

Reply:
Is `valueColor` appropriate? We dont want an animated color, right?

But I would suggest the following:
```dart
color: color,
```

Reply:
Acording to the offical documentation of flutter you can use new AlwaysStoppedAnimation<Color>()

Reply:
Yeah but please explain why to use an animated color here and then set it to a static color which is clearly what `color` is doing - overwriting the `valueColor` if set to null (by default)

Reply:
Is described like this in the Flutter documentation, but then I would leave it at that for now

Reply:
> Because it's a little bit hacky calling `Theme.of(context)` all the time
 
This is a technique often used in the native code of flutter, in this case I agree that it makes sense to just default to the `CircularProgressIndicator` color, but in general I would not consider it hacky

Reply:
It is hacky because there is a variant that does not require an extra call to be made.